src/main/java/duke/parser/Parser.java

```diff
@@ -0,0 +1,92 @@
+package duke.parser;
+
+import duke.command.*;
```

Comment:
Imported classes should always be listed explicitly. I noticed the same issue in serval other places too.

Reply:
Noted, thanks for the catch!